indicator_creation_rules/new_account_logging.py

```diff
@@ -0,0 +1,31 @@
+from datetime import timedelta
+
+from panther_oss_helpers import (
+    put_string_set,
+    set_key_expiration,
+    resolve_timestamp_string,
+)
+import panther_event_type_helpers as event_type
+
+# Days an account is considered new
+TTL = timedelta(days=3)
+
+
+def rule(event):
+    if event.udm("event_type") != event_type.ACCOUNT_CREATED:
+        return False
+
+    event_id = f'new_user_{event.get("p_row_id")}'
```

Comment:
What's the purpose of using the row ID  here? Is it just to have some value, or is it meaningful that this value is guaranteed to be unique?

Reply:
It's just to have a value. Initially I was going to use this as the key rather than the value until I realized how confusing that would be to search. Uniqueness isn't terribly important.